storage.go

```diff
@@ -4,33 +4,144 @@ import (
 	"context"
 	"io"
 
+	ipfs "github.com/ipfs/go-ipfs-api"
+
+	"github.com/beyondstorage/go-storage/v4/pkg/iowrap"
+	"github.com/beyondstorage/go-storage/v4/services"
 	. "github.com/beyondstorage/go-storage/v4/types"
 )
 
+// The src of `ipfs files cp` supports both `IPFS-path` and `MFS-path`
+// After `s.getAbsPath(src)`, if the absolute path matches `IPFS-path`, it will take precedence
+// This means that if the `workDir` is `/ipfs/`, there is a high probability that an error will be returned
+// See https://github.com/beyondstorage/specs/pull/134#discussion_r663594807 for more details
```

Comment:
This reminds me of a question: Do we need to support access objects directly via ipfs-path (CID)? e.g., `ipfs add/cat/pin`. (Will there be such a use case?)

Reply:
According to the design of IPFS, large files are partitioned into multiple blocks of up to 256KB each, and each block generates a corresponding CID by hash algorithm. 

The organization of multiple blocks is achieved by a Merkle DAG, which is found by the CID pointing to the head node.There is a high probability that the content accessed through the CID is not a complete file, which is often of no practical use to upper-level users, users with this level of granularity will consider using the IPFS-related libraries directly.

We can implement the function of accessing by any CID, but this is more similar to the `Block` operation, and a better approach may be to implement the block-related interface later.

Reply:
What about adding some descriptions (e.g., access via MFS) in the README/godoc?